modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/comments/PublisherCommentTest.java

```diff
@@ -224,80 +231,77 @@ In some databases(mysql, oracle), the timestamp format do not store milliseconds
             assertEquals(replyDTO.getParentCommentId(), rootCommentIdToAddReplies, "Comments do not match");
             assertEquals(replies.contains(replyDTO.getId()), true, "Comments do not match");
         }
+    }
 
-        // Verify Pagination of replies list of a comment
-        HttpResponse getRepliesToVerifyPagination = restAPIPublisher.getReplies(rootCommentIdToAddReplies, apiId, gatewayContextWrk
-                .getContextTenant().getDomain(), false, 3, 1);
+    @Test(dependsOnMethods = {"testPublisherGetRepliesOfCommentTest"}, groups = { "wso2.am" }, description = "Verify Pagination of Replies List of Comment Test Case")
+    public void testPublisherPaginationOfRepliesOfCommentTest() throws Exception {
+        HttpResponse getRepliesToVerifyPagination = restAPIPublisher
+                .getReplies(rootCommentIdToAddReplies, apiId, gatewayContextWrk.getContextTenant().getDomain(), false, 3, 1);
         assertEquals(getRepliesToVerifyPagination.getResponseCode(), Response.Status.OK.getStatusCode(),
                 "Error retrieving comment");
         Gson getRepliesToVerifyPaginationGson = new Gson();
-        CommentListDTO getRepliesToVerifyPaginationCommenListDTO = getRepliesToVerifyPaginationGson.fromJson(getRepliesToVerifyPagination
-                .getData().replace("publisher", "PUBLISHER"), CommentListDTO.class);
+        CommentListDTO getRepliesToVerifyPaginationCommenListDTO = getRepliesToVerifyPaginationGson
+                .fromJson(getRepliesToVerifyPagination.getData().replace("publisher", "PUBLISHER"), CommentListDTO.class);
         assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().size(), 2, "Comments limit does not match");
-        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(0).getContent(), "This is a reply 2", "Offset value does not captured");
-        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(1).getContent(), "This is a reply 3", "Offset value does not captured");
+        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(0).getContent(), "This is a reply 2",
+                "Offset value does not captured");
+        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(1).getContent(), "This is a reply 3",
+                "Offset value does not captured");
+    }
+
+    @DataProvider (name = "input-data-provider")
+    public Object[][] inputDataProviderMethod(){
```

Comment:
Shall we fix this formatting error?

Reply:
I will send this change with https://github.com/wso2/product-apim/pull/11163 PR

Reply:
Fixed with https://github.com/wso2/product-apim/pull/11163/commits/909e61ea2edc8a1e3f88aead2a5213324d94bd84